spring-cloud-huawei-sample/swagger-demo/edge-service/src/main/resources/microservice.yaml

```diff
@@ -0,0 +1,19 @@
+APPLICATION_ID: swagger
+service_description:
+  name: edge
+  version: 0.0.1
+  environment: development
+servicecomb:
+  service:
+    registry:
+      address: http://127.0.0.1:30100
```

Comment:
some code use ```https://cse.cn-east-2.myhuaweicloud.com```, some use ```http://127.0.0.1:30100```, why not use the same ?

Reply:
it will be fixed next pr